### PR TITLE
sequoia-sq 0.38.0 (new formula)

### PR DESCRIPTION
**Pull request by rrotter**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

```
~ % brew audit --new sequoia-sq
sequoia-sq
  * GitLab repository not notable enough (<30 forks and <75 stars)
Error: 1 problem in 1 formula detected.
```
https://gitlab.com/sequoia-pgp/sequoia-sq does not have very many stars, but I'd ask that you consider:
1. It's the main CLI for its library/parent project, https://gitlab.com/sequoia-pgp/sequoia, which has just under 400 stars
2. It's in Debian, Arch, and Fedora: https://repology.org/project/sequoia-sq/badges

### CODE REVIEW

File: Formula/s/sq.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ class Sq < Formula
 
   uses_from_macos "sqlite" => :test
 
-  conflicts_with "squirrel", because: "both install `sq` binaries"
```
Comment (chenrui333) [inline Formula/s/sq.rb:23]: these conflicts can be done as followup

File: Formula/s/sequoia-sq.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  desc "Sequoia-PGP command-line tool"
+  homepage "https://sequoia-pgp.org"
+  url "https://gitlab.com/sequoia-pgp/sequoia-sq/-/archive/v0.38.0/sequoia-sq-v0.38.0.tar.gz"
+  sha256 "9fd32ad0de3388804e21205003821710d2faf99d5c85a50bd97da3e7e480921b"
+  license "GPL-2.0-or-later"
+  head "https://gitlab.com/sequoia-pgp/sequoia-sq.git", branch: "main"
+
+  depends_on "capnp" => :build
+  depends_on "nettle" => :build
+  depends_on "rust" => :build
```
Comment (chenrui333) [inline Formula/s/sequoia-sq.rb:12]:
```suggestion
  depends_on "pkg-config" => :build
  depends_on "rust" => :build
```

File: Formula/s/sq.rb (hunk 1 above)
Comment (rrotter) [reply Formula/s/sq.rb]: Removed these from squirrel and sq. Should `conflicts_with` remain in in this formula?

File: Formula/s/sequoia-sq.rb (hunk 2 above)
Comment (rrotter) [reply Formula/s/sequoia-sq.rb:12]: Added this.

**Comment by rrotter** [comment]
Okay, there's clearly some issues to work out here. I'll try this in a Linux vm tomorrow evening. TBH I wrote this a while ago and really thought I'd tested on Linux too, but apparently I didn't.

**Comment by rrotter** [comment]
Okay, I've fixed the build failures (and made `brew linkage` happy afaict). Now the only error I'm getting is the `brew audit --new` notability failure. Can you consider an exception considering the mitigating factors I listed above?

**Comment by rrotter** [comment]
The last push was just a rebase to kick off a CI run. Still working up until the notability audit.

**Review by chenrui333 — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

**Comment by hakamadare** [comment]
hey @rrotter - thanks for this formula, you've inspired me to bundle up https://github.com/hakamadare/homebrew-sequoia-chameleon/pull/1 if you want to try that out.

**Pull request merged**